src/Monitor/Monitor/Alerts/NewAzureRmMetricAlertRuleV2CriteriaCommand.cs

```diff
@@ -61,78 +74,85 @@ public class NewAzureRmMetricAlertRuleV2CriteriaCommand : MonitorCmdletBase
         /// Gets or sets the rule condition operator
         /// </summary>
         [Parameter(Mandatory = true, HelpMessage = "The rule condition operator")]
-        [PSArgumentCompleter("GreaterThan", "GreaterThanOrEqual", "LessThan", "LessThanOrEqual")]
+        [PSArgumentCompleterWithScript(OperatorCompleterScript)]
```

Comment:
Potential Breaking Change: Operator parameter changed the completer type – if any values in the original completer were no longer valid, this would be a breaking change